src/app/screens/Calculator/index.js

```diff
@@ -2,12 +2,14 @@
 import React, { useState, useEffect } from 'react';
 import { connect } from 'react-redux';
 
+import OperationActions from 'redux/operations/actions';
+
 import CalculatorLayout from './layout';
 import { DIGITS, SIGNS, OPERATORS, OPERATORS_NOT_MINUS, SYN_ERR } from './constants.js';
 import styles from './styles.module.scss';
 import useMutableState from './hooks/useMutableState';
 
-const Calculator = () => {
+const Calculator = dispatch => {
```

Comment:
In general, when a component has an index.js file and a layout.js file, on the index.js file we name the component as `ComponentNameContainer` and in the Layout.js file we name the component just for it name. So here we should name this component `CalculatorContainer`